x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/session/Configuration.java

```diff
@@ -68,8 +78,11 @@ public EsqlConfiguration(
         assert tables != null;
     }
 
-    public EsqlConfiguration(BlockStreamInput in) throws IOException {
-        super(in.readZoneId(), Instant.ofEpochSecond(in.readVLong(), in.readVInt()), in.readOptionalString(), in.readOptionalString());
+    public Configuration(BlockStreamInput in) throws IOException {
+        this.zoneId = in.readZoneId();
+        this.now = Instant.ofEpochSecond(in.readVLong(), in.readVInt()).atZone(zoneId);
+        this.username = in.readOptionalString();
```

Comment:
This one is always `null` with a `TODO` next to it in `TransportEsqlQueryAction`.
In SQL we used this one for a function called `USER()` to return the current user. Same goes for `clusterName` - it was used for a function called `DATABASE()`.

Reply:
:+1: